### ISSUE TITLE
google home, everything "offline"

### ISSUE THREAD

**Issue by pixelion8** [t0bst4r/home-assistant-matter-hub#163]
Hi everyone, 🤗
thanks for this awsome project. I was able to install everything thanks to the readme being super clear. 👍 
Unfortunatly I have a small issue.

Issue :
After scanning the qr code all of my entity was correctly installed in my google home app (100+) but everything is grey and are marked "offline". So I can't use them.

My build :
nas synology -> docker home assistant + docker home-assistant-matter-hub
google home -> google home nest mini
philips hue lights, tapo plug, logitech harmony ...

Can you help me ? 😁
thanks

**Comment by t0bst4r** [comment]
First of all, can you test if it works with just a few entities. Let’s just test it with less than 10. 

Second, do you have any special network configuration? I have seen this problem before with VLAN setup.

Third, can you attach the log during the commissioning process?

**Comment by pixelion8** [comment]
1) I have put :
```
{
  "filter": {
    "include": [
      {
        "type": "pattern",
        "value": "light.top_salon_1"
      }
    ],
    "exclude": []
  }
}
```
So I only have one light in my bridge.
My google home app doesn't update and I still can see all of my entities (greyed out).
The only light in my bridge is still greyed out.

2) VLAN :
no I don't have that. My home assistant, my home assistant matter hub and my nas are on 192.168.1....
 3) sorry I don't know how to do that, can you enlight me ?

**Comment by t0bst4r** [comment]
> info : I checked my ipv6 on the website https://test-ipv6.com/ and everything is ok

i am not sure if that is sufficient. You need IPv6 enabled in your local network.
E.g. for me this means:
- internally i have IPv6 enabled - everything works
- My Internet Service Provider does not provide IPv6 (without extra costs) - `test-ipv6.com` claims, that it's not working

Technically this is correct, because between my Router and the webpage IPv4 is used.
Vice versa, it **could** be, that IPv4 is used internally, but IPv6 is used between Router and webpage, right?

Can you please check your router configuration, if IPv6 is enabled for internal network communication?

**Comment by Apollon77** [comment]
and because I saw you use docker. Please also make sure to have IPV6 enabled in docker and to use the host mode for networking. bridge mode is not working

**Comment by pixelion8** [comment]
ok thank you guys for your kindness. I managed to make it work. It was indeed an issue with ipv6 who had to be enabled in docker. The procedure for synology was a bit différent. I needed to connect through ssh and reach two places in order to activate ipv6 :
/var/packages/ContainerManager/etc/dockerd.json
and 
/volume1/@appstore/ContainerManager/config/dockerd.json
same name but two different files.
and add : 
"ipv6": true,"fixed-cidr-v6": "xxxx:xxxx:xxxx::/64"  // replace xxxx... by your FIX ipv6 gate (passerelle in french, fix ipv6 can be found in your internet provider router settings)

Of course, don't forget to activate ipv6 in your dsm (your nas settings).
I hope it will help someone else. Because, dammm it was hard for me 🥲
